.ci/Jenkinsfile

```diff
@@ -205,11 +205,11 @@ def generateFunctionalTestStep(Map params = [:]){
         unstash 'build'
         retry(3){
           dir("${BASE_DIR}"){
-            sh script: """.ci/scripts/install-test-dependencies.sh "${GO_VERSION}" """, label: "Install test dependencies for ${suite}:${feature}"
+            sh script: """.ci/scripts/install-test-dependencies.sh "${GO_VERSION}" "${suite}" """, label: "Install test dependencies for ${suite}:${feature}"
           }
         }
         dir("${BASE_DIR}"){
-          sh script: """.ci/scripts/functional-test.sh "${GO_VERSION}" "${suite}" "${feature}" "${STACK_VERSION}" "${METRICBEAT_VERSION}" """, label: "Run functional tests for ${suite}:${feature}"
```

Comment:
I'm removing the GO_VERSION from the `functional-test.sh` script because it is used to install Go, which is already done in the `install-test-dependencies.sh` script. And as the CI executes both in a decoupled way, I consider the 1st script is not self-contained. As a follow-up, we could include the `install-test-dependencies` script into the 1st script, becoming self-contained.

Reply:
Mmm, I think I see why both scripts are executed in a decoupled manner: to support retries installing things. We'd need then that the Go binary is still available to the second execution.